cmake/sca/codechecker/sca.cmake

```diff
@@ -5,6 +5,40 @@
 find_program(CODECHECKER_EXE NAMES CodeChecker codechecker REQUIRED)
 message(STATUS "Found SCA: CodeChecker (${CODECHECKER_EXE})")
 
+# Get CodeChecker specific variables
+zephyr_get(CODECHECKER_ANALYZE_JOBS)
```

Comment:
I know each SCA tool will have its own prefered way of being configured, but at the same time Zephyr users are familiar with Kconfig.

During review of Eclair integration it became clear that a better Kconfig integration to SCA tools is needed, see: #68324 and https://github.com/zephyrproject-rtos/zephyr/issues/77018, especially in a way that the SCA Kconfig is only loaded when the tool is available.

The list of settings below looks like something that would be better placed in Kconfig, and thereby also be auto documented using the Kconfig help text.

Perhaps with an exception for the `CODECHECKER_CONFIG_FILE` as that could look like something one might want to pass in directly on CMake invocation.

Thoughts ?